doc/march_packages/march_gait_selection.rst

```diff
@@ -28,6 +28,21 @@ of an old and new gait name. This structure requires that both gait names have m
 transition. Using the two subgait trajectories a new transition trajectory is calculated and stored as a subgait object.
 If this is calculated correctly, the new subgait trajectory is sent to the march_gait_scheduler.
 
+Balance walk
+^^^^^^^^^^^^
+The gait selection packages is currently also the package were we store dynamic gaits. One such dynamic gait is the
+balance walk gait. The graph of the balance walk gait is defined in the :ref:`march-gait-files-label`.
+Some of the subgaits in this gait are just static subgaits defined by a subgaitfile. However, some of the subgaits are dynamic.
+In these subgaits we try to move towards a capture point. A capture point is a point on the ground such that if you would
+place your foot there, you would come to a stall. There are a lot of papers on the internet about this principle.
```

Comment:
Not sure if [_stall_](https://www.merriam-webster.com/dictionary/stall) is the correct word to use here

Reply:
haha :  a compartment for a domestic animal in a stable or barn